### ISSUE TITLE
Asyncio has no attribute 'ensure_future' AWS Lambda output.  Zappa/AWS Python3.6 

### ISSUE THREAD

**Issue by XhrCkYsSgZokJL** [Miserlou/Zappa#1362]
When deploying a project which uses the asyncio library taking advantage of asyncio.ensure_future(), zappa tail outputs: 
AttributeError: module 'asyncio' has no attribute 'ensure_future'

## Expected Behavior
For the asyncio tasks to complete without any issues. 

## Actual Behavior

> [1516740487026] [2018-01-23 20:48:07,025] ERROR in app: Exception on / [GET]
> [1516740487026] Traceback (most recent call last):
> [1516740487026] File "/var/task/flask/app.py", line 1982, in wsgi_app
> [1516740487026] response = self.full_dispatch_request()
> [1516740487026] File "/var/task/flask/app.py", line 1614, in full_dispatch_request
> [1516740487026] rv = self.handle_user_exception(e)
> [1516740487026] File "/var/task/flask/app.py", line 1517, in handle_user_exception
> [1516740487026] reraise(exc_type, exc_value, tb)
> [1516740487026] File "/var/task/flask/_compat.py", line 33, in reraise
> [1516740487026] raise value
> [1516740487026] File "/var/task/flask/app.py", line 1612, in full_dispatch_request
> [1516740487026] rv = self.dispatch_request()
> [1516740487026] File "/var/task/flask/app.py", line 1598, in dispatch_request
> [1516740487026] return self.view_functions[rule.endpoint](**req.view_args)
> [1516740487026] File "/var/task/index.py", line 27, in doSomething
> [1516740487026] conversionTasks.append(asyncio.ensure_future(getSomething(example)))
> [1516740487026] AttributeError: module 'asyncio' has no attribute 'ensure_future'

## Possible Fix
Is this an issue with AWS Lambda not applying python3.6?  Zappa not enforcing it? 

## Steps to Reproduce
```
from flask import Flask
from flask import request
import simplejson as json
import asyncio
import aiohttp

app = Flask(__name__)

async def getSomething(example):
	return ''

@app.route('/')
def doSomething():
	loop = asyncio.get_event_loop()
	response = {}

	conversionTasks = []
	example = {"x": "y"}
	conversionTasks.append(asyncio.ensure_future(getSomething(example)))
	conversionResults = loop.run_until_complete(asyncio.gather(*conversionTasks))

	return json.dumps(response)

if __name__ == '__main__':
	app.run()
```

## Your Environment
Zappa: 0.45.1

zappa_settings.json
```
{
    "dev": {
        "app_function": "index.app",
        "aws_region": "us-east-2",
        "profile_name": "default",
        "project_name": "tests_0043",
        "runtime": "python3.6",
        "s3_bucket": "zappa-########"
    }
}
```

pip3 freeze

```
aiohttp==2.3.9
argcomplete==1.9.2
async-timeout==2.0.0
asyncio==3.4.3
base58==0.2.4
boto3==1.5.20
botocore==1.8.34
certifi==2018.1.18
cfn-flip==1.0.0
chardet==3.0.4
click==6.7
docutils==0.14
durationpy==0.5
Flask==0.12.2
future==0.16.0
hjson==3.0.1
idna==2.6
itsdangerous==0.24
Jinja2==2.10
jmespath==0.9.3
kappa==0.6.0
lambda-packages==0.19.0
MarkupSafe==1.0
marshmallow==2.15.0
multidict==4.0.0
placebo==0.8.1
python-dateutil==2.6.1
python-slugify==1.2.4
PyYAML==3.12
requests==2.18.4
s3transfer==0.1.12
simplejson==3.13.2
six==1.11.0
toml==0.9.4
tqdm==4.19.1
troposphere==2.1.2
Unidecode==1.0.22
urllib3==1.22
webargs==1.8.1
Werkzeug==0.12
wsgi-request-logger==0.4.6
yarl==1.1.0
zappa==0.45.1
```

**Comment by scoates** [comment]
I noticed you did `pip3 freeze`. Just want to be sure that:

- the python in your active virtualenv is 3.6, right?
- the deployed Zappa stage's Lambda is and was always 3.6, not trying to upgrade from a previously deployed 2.7 or anything like that?

**Comment by kybrdbnd** [comment]
facing the same issue !!! help required

**Comment by XhrCkYsSgZokJL** [comment]
Seemingly issues because I wasn't using venv for everything.  Closing for now.

**Comment by kirbyfu** [comment]
Was facing a similar issue. Eventually realized that asyncio was included in the python stdlib (since Python 3.4.4) so I removed it from `requirements.txt`. No issues after that.

**Comment by markbulgakov** [comment]
you need asyncio version not less than 3.4.4
